synapseclient/api/wiki_service.py

```diff
@@ -0,0 +1,456 @@
+"""This module is responsible for exposing the services defined at:
+<https://rest-docs.synapse.org/rest/#org.sagebionetworks.repo.web.controller.WikiController2>
+"""
+
+import json
+from typing import TYPE_CHECKING, Any, AsyncGenerator, Dict, List, Optional
+
+if TYPE_CHECKING:
+    from synapseclient import Synapse
+
+
+async def post_wiki(
+    owner_id: str,
+    request: Dict[str, Any],
+    *,
+    synapse_client: Optional["Synapse"] = None,
+) -> Dict[str, Any]:
+    """Create a new wiki page.
+    <https://rest-docs.synapse.org/rest/POST/entity/ownerId/wiki2.html>
+
+    Arguments:
+        owner_id: The ID of the owner entity.
+        request: The wiki page to create.
+        synapse_client: If not passed in and caching was not disabled by
+                `Synapse.allow_client_caching(False)` this will use the last created
+                instance from the Synapse class constructor.
+
+    Returns:
+        The created wiki page.
+    """
+    from synapseclient import Synapse
+
+    client = Synapse.get_client(synapse_client=synapse_client)
+    return await client.rest_post_async(
+        uri=f"/entity/{owner_id}/wiki2",
+        body=json.dumps(request),
+    )
+
+
+async def get_wiki_page(
+    owner_id: str,
+    *,
+    wiki_id: Optional[str] = None,
```

Comment:
Nit: Adding these as kwargs instead of positional args is inconsistent with the other functions you have.